_includes/tutorials/aggregating-average/kstreams/code/docker-compose.yml

```diff
@@ -0,0 +1,60 @@
+---
+version: '3.5'
+
+services:
+  zookeeper:
+    image: confluentinc/cp-zookeeper:5.4.0
```

Comment:
We can go ahead and bump CP versions to 5.4.1

Reply:
done